### PR TITLE
Multi subgraphs

### PR DESCRIPTION
**Pull request by ramabit**
# Description

Generating separated subgraphs for dev, staging and prod in order to use and test different queries on different envs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ const sdkInstance = new TheBadgeSDK(
     // read only provider
     rpcProviderConfig: {
       // your desired provider: 'infura' or 'alchemy'
-      name: RPCProviders.infura,
+      name: 'infura',
     
       // your provider token
       token: '00000000000000',
```
Comment (Agupane) [inline README.md:46]: this should be replace by apiKey

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ const sdkInstance = new TheBadgeSDK(
     // read only provider
     rpcProviderConfig: {
       // your desired provider: 'infura' or 'alchemy'
-      name: RPCProviders.infura,
+      name: 'infura',
     
       // your provider token
```
Comment (Agupane) [inline README.md:45]: this should be:
```your infura/alchemy apiKey```

**Review by Agupane — CHANGES_REQUESTED** [review]
Lets use this opportunity to update some docs on the readme

File: README.md (hunk 1 above)
Comment (ramabit) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (ramabit) [reply README.md]: done

**Review by Agupane — APPROVED** [review]

**Pull request merged**